.github/workflows/main.yml

```diff
@@ -47,9 +47,9 @@ jobs:
         run: |
           set -e
           check_dir() { # $1 is the file extension, $2 is the directory and $3 the max depth to check
-            result=$(find $2 -maxdepth ${3:-1000} -name "$1" -type f -exec egrep -l " +$" {} \;)
+            result=$(find $2 -maxdepth ${3:-1000} -name "$1" -type f -exec grep -E -Hn " +$" {} \;)
             if [ -n "$result" ]; then
-              echo "Trailing whitespace found in the following files:"
+              echo "Trailing whitespace found in the following locations:"
```

Comment:
When I added this check, I found it too noisy when there were many locations with trailing whitespace and editors can usually trims them automatically so I thought it was redundant.